webapp/src/components/expanded_view/reaction_button.tsx

```diff
@@ -177,6 +177,25 @@ export const ReactionButton = forwardRef(({trackEvent}: Props, ref) => {
     );
 });
 
+const findEmojiName = (names: string[]) => {
```

Comment:
Do we know how complex this is? (time/space wise) Assuming this is executed every time someone sends a reaction just want to make sure we are not introducing a performance bottleneck.

Reply:
Everytime they click an emoji, roughly O(N), where N is number of names in the list (from around 1 to 3) : ~1-3 names per list, mapping over those (string and array garbage) and then those lookups in a static map. I considered doing a two stage (like search in the map first, then if not found do the mapping, then lookup again), but I really don't think this is going to be an impact. 
I would maybe be concerned if it was an automated thing, but this is only per emoji click.

Reply:
Sounds good, thanks :+1: